lib/workers/repository/process/lookup/__snapshots__/index.spec.ts.snap

```diff
@@ -175,12 +175,10 @@ Object {
 
 exports[`workers/repository/process/lookup/index .lookupUpdates() handles git submodule update 1`] = `
 Object {
-  "currentVersion": undefined,
```

Comment:
not sure if we expect those changes

Reply:
Hah, this is caused by `valueToVersion` removal, so new snapshot seems to be more valid